cpp/oneapi/dal/algo/svm/train_types.cpp

```diff
@@ -102,7 +102,13 @@ const table& train_result<Task>::get_coeffs() const {
 
 template <typename Task>
 double train_result<Task>::get_bias() const {
-    return impl_->trained_model.get_bias();
+    // return dal::detail::get_impl(impl_->trained_model).bias;
+    return 0.0;
```

Comment:
Implementation should be valid despite this function is deprecated.

Reply:
done